src/components/Layout/Layout.stories.tsx

```diff
@@ -0,0 +1,93 @@
+import { Meta } from "@storybook/react";
+import { NestedScrollLayoutContextProvider } from "src/components/Layout/NestedScrollLayoutContext";
+import { ScrollableContent } from "src/components/Layout/ScrollableContent";
+import { TestHeader, TestLayout, TestPageContent, TestSideNav } from "src/components/Layout/TestLayout";
+import { Tab, Tabs } from "src/components/Tabs";
+import { Css } from "src/Css";
+import { noop } from "src/utils";
+import { withBeamDecorator, withDimensions, zeroTo } from "src/utils/sb";
+
+export default {
+  component: NestedScrollLayoutContextProvider,
+  title: "Components/NestedScrollLayoutContext",
+  decorators: [withBeamDecorator, withDimensions()],
+  parameters: { layout: "fullscreen" },
+} as Meta;
+
+export function BasicLayout() {
+  return (
+    <TestLayout>
+      <ExamplePageComponent />
+    </TestLayout>
+  );
+}
+
+export function ContentAboveTable() {
+  return (
+    <TestLayout>
+      <ExamplePageComponent contentAboveTable />
+    </TestLayout>
+  );
+}
+
+export function SideNavLayout() {
+  return (
+    <TestLayout>
+      <TestSideNav />
+      <ExamplePageComponent />
+    </TestLayout>
+  );
+}
+
+function ExamplePageComponent({ contentAboveTable }: { contentAboveTable?: boolean }) {
+  const tabs: Tab[] = [
+    { value: "overview", name: "Overview", render: () => <></> },
+    { value: "lineItems", name: "Line Items", render: () => <></> },
+    { value: "history", name: "History", render: () => <></> },
+  ];
+  return (
+    <NestedScrollLayoutContextProvider>
```

Comment:
I'm thinking this would move up one more level and probably be in like a `PageLayout` or `ProjectLayout` component. That way not _every_ page needs to remember to wrap itself in this... though every page would need to remember to wrap itself in `<ScrollableContent />` where applicable.